src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,16 @@
 package mate.academy
 
+import mate.academy.model.Figure
+import mate.academy.service.FigureSupplier
+
+const val ARRAY_SIZE = 4
+
 fun main() {
     // draw all figures here
+    val figureSupplier = FigureSupplier()
+    val arrayMedian = ARRAY_SIZE / 2
+    val figures: Array<Figure> = Array(ARRAY_SIZE) { i ->
+        if (i < arrayMedian) figureSupplier.getRandomFigure()
```

Comment:
Please check formatting. You may use CTRL+ALT+L.
Also it is better to use {} brackets  for if/else statements